beater/java_attacher/java_attacher.go

```diff
@@ -55,6 +54,9 @@ func New(cfg config.JavaAttacherConfig) (JavaAttacher, error) {
 		cfg.JavaBin = filepath.FromSlash(cfg.JavaBin)
 	}
 	logger := logp.NewLogger("java-attacher")
+	if _, err := os.Stat(javaAttacher); err != nil {
```

Comment:
this was an initial error I encountered, the error output from when the attacher was missing was super-unhelpful. The stat error, `failed to start java attacher: stat ./java-attacher.jar: no such file or directory`, is much more helpful.